client/CHANGELOG.md

```diff
@@ -24,15 +24,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - `alias_id_to_bech32()` to utils;
+- `Client::finish_block_builder()`;
 
 ### Changed
 
 - Abort the sync nodes task through its handle instead of a oneshot channel;
 - Update network info also from unhealty nodes if ignore_node_health is set to true;
+- `finish_pow()` and `do_pow()` now accept optional parents and always do PoW, independent of the local PoW setting in Client;
```

Comment:
We could make them private since `finish_block_builder()` does everything, but not sure if someone might still want to  have them